app/src/main/kotlin/com/ayatk/biblio/util/RxUtils.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2018 ayatk.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ayatk.biblio.util
+
+import io.reactivex.Maybe
+import io.reactivex.Single
+
+fun <T : Any> T.toSingle(): Single<T> = Single.just(this)
+fun <T : Any> T?.toMaybe(): Maybe<T> = Maybe.create { s -> if (this != null) s.onSuccess(this); s.onComplete() }
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
This if block is empty and can be removed.